layer-atlas/src/main/java/com/layer/atlas/adapters/AtlasConversationsAdapter.java

```diff
@@ -238,6 +247,41 @@ public void onBindViewHolder(ViewHolder viewHolder, int position) {
         }
     }
 
+    private void setAvatar(ViewHolder viewHolder, Set<Identity> participants) {
+        if (participants.size() > 0) {
+            Identity participant = participants.iterator().next();
+            if (participant == null) {
+                return;
+            }
+            if (TextUtils.isEmpty(participant.getAvatarImageUrl())) {
```

Comment:
I think these three cases can be replaced with just one picasso call with `error` and `placeholder`

Reply:
👍 true, I've checked the documentation and null is handled by Picasso :)